### DISCUSSION TITLE
unmap keybinding

### DISCUSSION THREAD

**Discussion by realyukii** [philc/vimium#4527] [General]
I would like to know if it's possible to unbind a specific key on a particular website.

For example,
- on GitHub, I want to unmap the `/` key to make the GitHub search functionality work.
- `f` key conflict with full screen key on youtube
- ~~I want to unmap `<C-e>` and `<C-y>` on all pages because I mainly use `hjkl` for scrolling. The `<C-e>` shortcut interferes with my ability to navigate to the URL bar.~~

EDIT: I've just read `unmap` on readme, update question: "how to unmap on specific site"

**Comment by tjex** [comment]
I can't see an option for this. I would recommend just remapping to something else similar for search / find functionality, like `<c-f>`, or perhaps `<c-/>`

**Reply by realyukii** [reply]
Thanks for the suggestion! I will apply it. appreciate this extension, very awesome and make my browsing experience very convenient :)
